addons/mail/models/mail_message.py

```diff
@@ -883,6 +883,38 @@ def set_message_done(self):
             },
         )
 
+    def move_to_inbox(self):
```

Comment:
Should be "mark_unread", which is the real model action we perform. Probably already existed at some point by the way, remember we had a method like that as it was possible to set messages as unread. Not sure why it does not exist anymore.